### PR TITLE
Use specific commit rather than branch

### PR DESCRIPTION
**Pull request by PwzXxm**

### CODE REVIEW

**Comment by liliu-z** [comment]
Let use a short one next time

File: cmake/libs/libcardinal.cmake [hunk 1]
```diff
@@ -1,4 +1,4 @@
-set(CARDINAL_VERSION master)
+set(CARDINAL_VERSION 0bb6d5771a7c279362bd3b6ea7dc94b53d0c4de1 )
```
Comment (foxspy) [inline cmake/libs/libcardinal.cmake:1]: use short commit tag?

**Pull request merged**